core/google/cloud/exceptions.py

```diff
@@ -20,18 +20,27 @@
 # Avoid the grpc and google.cloud.grpc collision.
 from __future__ import absolute_import
 
+import contextlib
 import copy
 import json
-import six
-
-from google.cloud._helpers import _to_bytes
+import sys
 
-_HTTP_CODE_TO_EXCEPTION = {}  # populated at end of module
+import six
 
 try:
+    from google.gax.errors import GaxError
+    from google.gax.grpc import exc_to_code
+    from grpc import StatusCode
     from grpc._channel import _Rendezvous
 except ImportError:  # pragma: NO COVER
+    _HAVE_GRPC = False
     _Rendezvous = None
+else:
+    _HAVE_GRPC = True
```

Comment:
This is what `else:` is for:  assigning after the failing import in the `try:` is much uglier.